spec/features/manage_cmr/subscriptions/test_query_spec.rb

```diff
@@ -17,7 +17,7 @@
   before do
     login
 
-    allow_any_instance_of(SubscriptionPolicy).to receive(:create?).and_return(true)
+    allow_any_instance_of(SubscriptionPolicy).to receive(:update?).and_return(true)
     allow_any_instance_of(SubscriptionPolicy).to receive(:show?).and_return(true)
```

Comment:
I think both of these mocked permissions will be provided by the before all block.